fixtures/proc-macro/src/lib.rs

```diff
@@ -270,4 +270,9 @@ fn get_externals(e: Option<Externals>) -> Externals {
     e.unwrap_or_default()
 }
 
+#[uniffi::export]
+pub fn join(parts: &[String], sep: &str) -> String {
```

Comment:
Sadly, still no support for `&[&str]`, that would require a lot more work I think.